tests/devices.spec.ts

```diff
@@ -106,3 +106,61 @@ test("profile edit networks", async ({ page }) => {
 
   await deleteProfile(page, profile);
 });
+
+test("add, edit and remove a proxy device", async ({ page }) => {
+  const profile = randomProfileName();
+
+  //Add Proxy Device
+  await startProfileCreation(page, profile);
+  await page.getByText("Proxy", { exact: true }).click();
+  await page.getByRole("button", { name: "New Proxy Device" }).click();
+  await page.getByLabel("Bind").selectOption({ index: 2 });
+  await page.getByPlaceholder("127.0.0.1").first().fill("127.0.0.1");
+  await page.getByPlaceholder("[-00]").first().fill("3000");
+  await page
+    .locator('[id="devices\\.0\\.connect\\.address"]')
+    .fill("127.0.0.2");
+  await page.getByPlaceholder("[-00]").nth(1).fill("3001");
+  await finishProfileCreation(page, profile);
+
+  await visitProfile(page, profile);
+  await page.getByTestId("tab-link-Configuration").click();
+  await page.getByText("Proxy", { exact: true }).click();
+  await page
+    .getByRole("gridcell", { name: "127.0.0.1" })
+    .getByPlaceholder("127.0.0.1")
+    .fill("127.0.0.3");
+  await saveProfile(page, profile, 1);
+
+  //Edit and save Proxy Device
+  await visitProfile(page, profile);
+  await page
+    .getByRole("gridcell", { name: "127.0.0.3" })
+    .getByPlaceholder("127.0.0.1")
+    .isVisible();
```

Comment:
I think we should get by the label here as well, to ensure the right field has received the right value. Also Better to use expect, like below. This is true also for the other assertions below.

```
await expect(
    page....,
  ).toBeVisible();
```